src/main/js/utils/Schedule.js

```diff
@@ -32,7 +32,16 @@ class Schedule extends Set {
       const name = `${course.subject}${course.courseNumber}`;
       const instructor = getInstructorLastName(course.instructor);
 
-      if (!gradeMap[name] || !gradeMap[name][instructor]) return;
+      if (!gradeMap[name]) {
+        return;
+      }
+
+      if (!gradeMap[name][instructor] && !gradeMap[name].Staff) return;
+      if (!gradeMap[name][instructor] && gradeMap[name].Staff) {
+        qualityCredits += gradeMap[name].Staff * course.credits;
+        totalCredits += course.credits;
+        return;
+      }
```

Comment:
Instead of doing this, pass in `useCourseAvg`. To get access to it in `CourseTable` you'll need another value in state (like you did for the checkbox) but don't make another action/reducer. Just add the value to the initial state and add a second parameter to the existing action/reducer.

Reply:
Fixed.